dashboard/models.py

```diff
@@ -150,13 +151,46 @@ def get_supported_courses(self):
             logger.info("Courses did not exist", exc_info = True)
         return []
```

Comment:
This isn't part of the change but I think this needs to return an empty QuerySet. Should probably add a typehint here. 

It sounds like the best way to do it is
Course.objects.none()